### PR TITLE
try to fix for new data layout

### PR DESCRIPTION
**Pull request by GaelVaroquaux**

### CODE REVIEW

**Comment by emmanuelle** [comment]
~~Looks good to me, thanks a lot!~~ EDIT : there was still a problem

File: fetcher.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 FILENAME_JOHN_HOPKINS = {
     "confirmed": "time_series_covid19_confirmed_global.csv",
     "death": "time_series_covid19_deaths_global.csv",
```
Comment (emmanuelle) [inline fetcher.py]: the global files have a line for the United States (name is US)

**Comment by emmanuelle** [comment]
@GaelVaroquaux please check when the artifact is up

**Comment by GaelVaroquaux** [comment]
Looks good! It was simpler than what I had tried.

Thanks! Merging!

**Pull request merged**